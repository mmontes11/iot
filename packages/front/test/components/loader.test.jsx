import React from "react";
import { shallow } from "enzyme";
import Loader from "components/loader";

describe("components/loader", () => {
  it("renders", () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper).toMatchSnapshot();
  });
});
