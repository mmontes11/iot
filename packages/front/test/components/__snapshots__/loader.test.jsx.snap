// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/loader renders 1`] = `
<div
  className="is-horizontal-center"
>
  <figure
    className="image is-128x128"
  >
    <img
      alt="Loader"
      src="test-file-stub"
    />
  </figure>
</div>
`;
